easybuild/easyconfigs/l/libsodium/libsodium-1.0.18-GCCcore-12.3.0.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsodium'
+version = '1.0.18'
+
+homepage = 'https://doc.libsodium.org/'
+description = """
+ Sodium is a modern, easy-to-use software library for encryption, decryption,
+ signatures, password hashing and more.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://download.%(name)s.org/%(name)s/releases/',
+    'https://download.%(name)s.org/%(name)s/releases/old/',
+    'https://download.%(name)s.org/%(name)s/releases/old/unsupported/',
+]
+sources = [SOURCE_TAR_GZ]
+checksums = ['6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1']
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+
+sanity_check_paths = {
+    'files': ['include/sodium.h', 'lib/%(name)s.%s' % SHLIB_EXT, 'lib/%(name)s.a'],
```

I am really sorry for make a wrong suggestion earlier. This should be the correct syntax:
```suggestion
    'files': ['include/sodium.h', 'lib/%%(name)s.%s' % SHLIB_EXT, 'lib/%(name)s.a'],
```